scm/driver/fake/pr.go

```diff
@@ -206,6 +232,13 @@ func (s *pullService) UnrequestReview(ctx context.Context, repo string, number i
 
 func (s *pullService) Create(_ context.Context, fullName string, input *scm.PullRequestInput) (*scm.PullRequest, *scm.Response, error) {
 	f := s.data
+
+	for _, pullRequest := range f.PullRequests {
```

Comment:
Not sure how big f.PullRequests is, but it's preferred to use the index instead of the value
Normally our linters (go-critic) will complain abt it.
So something like this:
```
for k := range f.PullRequests {
  pullRequest = f.PullRequests[i]
}
```